contracts/andromeda_anchor/src/contract.rs

```diff
@@ -181,7 +185,7 @@ pub fn withdraw(
                 contract_addr: env.contract.address.to_string(),
                 msg: to_binary(&ExecuteMsg::Yourself {
                     yourself_msg: YourselfMsg::TransferUst {
```

Comment:
Can we avoid this by making the messages to Anchor be submessages rather than regular messages?